scripts/qa_crowbarsetup.sh

```diff
@@ -491,10 +492,17 @@ function onadmin_prepare_sles_repos()
         add_mount "SUSE-Cloud-SLE-11-SP3-deps/sle-11-x86_64/" "" "${targetdir_install}" "Cloud-Deps"
         zypper_refresh
     else
-        zypper se -s sles-release|grep -v -e "sp.up\s*$" -e "(System Packages)" |grep -q x86_64 || zypper ar http://$susedownload/install/SLP/SLES-${slesversion}-LATEST/x86_64/DVD1/ sles
+        zypper se -s sles-release | \
+            grep -v -e "sp.up\s*$" -e "(System Packages)" | \
+            grep -q x86_64 \
+        || zypper ar \
+            http://$susedownload/install/SLP/SLES-${slesversion}-LATEST/x86_64/DVD1/ \
+            sles
 
         if ! $longdistance ; then
-            add_mount "" "clouddata.cloud.suse.de:/srv/nfs/suse-$suseversion/install" "${targetdir_install}"
+            add_mount "" \
```

Comment:
The counter-example does not cancel out the fact that long lines cause breakage in some places, so it's not really relevant.  And even if github handled long lines perfectly, exceeding 80-column lines is still a) making life harder for other people and b) against the conventional wisdom of most of the industry.  You wouldn't try to submit a patch with long lines to an OpenStack project, so why do it here?